junebug/tests/test_api.py

```diff
@@ -25,6 +29,10 @@ def get(self, url, params={}):
         return treq.get(
             "%s%s" % (self.url, url), params=params, persistent=False)
 
+    def request(self, method, url, params={}):
+        return treq.request(
+            method, "%s%s" % (self.url, url), params=params, persistent=False)
```

Comment:
I'm not sure that we need this function, we only seem to be using the `GET` method, and that functionality is fully covered by the `get` function above.

Reply:
Ah okay, I see 👍